libcxx/include/__functional/function.h

```diff
@@ -641,28 +504,6 @@ class __policy_func<_Rp(_ArgTypes...)> {
 public:
```

Comment:
On old line 586: `// __policy_invoker calls an instance of __alloc_func held in __policy_storage.`